cacerts/jksutil/jks.go

```diff
@@ -0,0 +1,183 @@
+// package jksutil allows interactions with a version 2 java key store
```

Comment:
Please add the license header found [here](https://github.com/GoogleContainerTools/distroless/blob/master/base/testdata/check_certs.go) (with year 2020).